app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/apps/TrackingProtectionAppsRepository.kt

```diff
@@ -192,19 +195,13 @@ class RealTrackingProtectionAppsRepository @Inject constructor(
 
     override suspend fun isAppProtectionEnabled(packageName: String): Boolean {
         Timber.d("TrackingProtectionAppsRepository: Checking $packageName protection status")
+        val appInfo = packageManager.getApplicationInfo(packageName, 0)
         val appExclusionList = appTrackerRepository.getAppExclusionList()
         val manualAppExclusionList = appTrackerRepository.getManualAppExclusionList()
 
-        val userExcludedApp = manualAppExclusionList.find { it.packageId == packageName }
-        if (userExcludedApp != null) {
-            return userExcludedApp.isProtected
-        }
-
-        if (appTrackerRepository.getSystemAppOverrideList().map { it.packageId }.contains(packageName)) {
-            return true
-        }
+        val isExcluded = shouldBeExcluded(appInfo, appExclusionList, manualAppExclusionList)
 
-        return !appExclusionList.any { it.packageId == packageName }
```

Comment:
Incidentally fixed this method that was causing a bug where games would always appear protected in the App tracking protection app detailed screen